run.py

```diff
@@ -0,0 +1,31 @@
+import torch
+import typer
+from pathlib import Path
+
+import settings
+import model
+from inference import INF
+
+
+def main(
+    path: Path = typer.Argument(
+        ...,
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
+        readable=True,
+        resolve_path=True,
+        help="Path to file with fens",
+    )
+):
+    test = INF(
+        torch.device("cpu"),
+        model.Coder(settings.BOARD_SHAPE, settings.LATENT_SIZE).to(settings.DEVICE),

```

Comment:
I think it's worth adding some information on how to select the model location and where to get an example pretrained model (and maybe some example invocation, for example how to get embeddings for some example FEN)